docs/guides/dkms.md

```diff
@@ -134,6 +134,30 @@ It unloads the Touchbar modules as they can cause issues for suspend.
 
 Your keyboard backlight may remain switched off on resuming and backlight controls may stop working. A restart fixes the backlight controls. You may also run `echo 60 > /sys/class/leds/apple::kbd_backlight/brightness` to turn on the backlight to the maximum level if you do not want to boot. Replace 60 with a lower number for lower brightness.
 
+A possible workaround is to bind the backlight controls directly to the kbd_backlight by using `acpilight`.
+
+First add your user to the `video` group by running `sudo useradd -aG $USER video`
```

Comment:
```
aditya@MacBook:~$ sudo useradd -aG $USER video
[sudo] password for aditya: 
useradd: invalid option -- 'a'
```

I am getting this error.

Reply:
It’s supposed to be `usermod`, sorry for my part.